src/components/nav/nav.tsx

```diff
@@ -0,0 +1,201 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Host,
+  Prop,
+  State,
+  VNode
+} from "@stencil/core";
+import { slotChangeHasAssignedElement } from "../../utils/dom";
+import { CSS, LEVEL, SLOTS } from "./resources";
+
+type Level = "primary" | "secondary" | "tertiary";
+
+@Component({
+  tag: "calcite-nav",
+  styleUrl: "nav.scss",
+  shadow: {
+    delegatesFocus: true
+  }
+})
+
+/**
+ * @slot logo - A slot for adding a `calcite-logo` component to the primary nav level.
+ * @slot user - A slot for adding a `calcite-user` component to the primary nav level.
+ * @slot progress - A slot for adding a `calcite-progress` component to the primary nav level.
+ * @slot menu-action - A slot for adding a `calcite-action` component to the primary nav level.
+ * @slot primary-content-start - A slot for adding a `calcite-menu`, `calcite-action`, or other interactive elements in the start position of the primary nav level.
+ * @slot primary-content-center - A slot for adding a `calcite-menu`, `calcite-action`, or other interactive elements in the center position of the primary nav level.
+ * @slot primary-content-end - A slot for adding a `calcite-menu`, `calcite-action`, or other interactive elements in the end position of the primary nav level.
+ * @slot secondary-content-start - A slot for adding a `calcite-menu`, `calcite-action`, or other interactive elements in the start position of the secondary nav level.
+ * @slot secondary-content-end - A slot for adding a `calcite-menu`, `calcite-action`, or other interactive elements in the end position of the secondary nav level.
+ * @slot tertiary-content-start - A slot for adding a `calcite-menu`, `calcite-action`, or other interactive elements in the start position of the tertiary nav level.
+ * @slot tertiary-content-end - A slot for adding a `calcite-menu`, `calcite-action`, or other interactive elements in the end position of the tertiary nav level.
+ */
+export class CalciteNav {
+  //--------------------------------------------------------------------------
+  //
+  //  Element
+  //
+  //--------------------------------------------------------------------------
+  @Element() el: HTMLCalciteNavElement;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Properties
+  //
+  //--------------------------------------------------------------------------
+
+  /**
+   * When true, display a hamburger `calcite-action` and emits `calciteNavMenuActionSelect` event on user interaction.
```

Comment:
```
When `true`, displays a hamburger icon and emits a `calciteNavMenuActionSelect` event on selection change.
```

Reply:
based on the feedback [here](https://github.com/Esri/calcite-components/pull/6873#discussion_r1179656339) changed the hamburger icon to hamburger `calcite-action` . 

cc: @driskull

Reply:
Can we not use `hamburger`? I know it is the icon name but maybe just "displays a `calcite-action`" ?

Reply:
I'm on board with removing the hamburger reference, if Anvesh and Matt agree. It seems a bit confusing just as `calcite-action` though, what would folks think about (a similar change would need to occur on the `label` prop for consistency as well):

```
When `true`, displays a `calcite-action` menu and emits a `calciteNavMenuActionSelect` event on selection change.
```

Reply:
> ```
> When `true`, displays a `calcite-action` menu and emits a `calciteNavMenuActionSelect` event on selection change.
> ```

This sounds good to me. Wanted to confirm its `calciteNavActionSelect` not `calciteNavMenuActionSelect` .